src/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/RijndaelImplementation.cs

```diff
@@ -0,0 +1,87 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
+{
+    /// <summary>
+    /// Internal implementation of Rijndael.
+    /// This class is returned from Rijndael.Create() instead of the public RijndaelManaged to 
+    /// be consistent with the rest of the static Create() methods which return opaque types.
+    /// </summary>
+    internal sealed class RijndaelImplementation : Rijndael
+    {
+        private readonly Aes _impl;
+
+        internal RijndaelImplementation()
+        {
+            LegalBlockSizesValue = new KeySizes[] { new KeySizes(minSize: 128, maxSize: 128, skipSize: 0) };
+
+            // This class wraps Aes
+            _impl = Aes.Create();
+            _impl.BlockSize = 128;
```

Comment:
Since AES only supports 128 this line is a bit wasteful.  If you wanted it as an assert that might be reasonable for future-proofing.